apistar/apischema.py

```diff
@@ -14,16 +14,20 @@
 from apistar.routing import (
     Route, RoutesConfig, primitive_types, schema_types, walk
 )
+from apistar.settings import Settings
 from apistar.templating import Templates
 
 
 class APISchema(Document):
     @classmethod
-    def build(cls, app: App, base_url: http.URL=None):
+    def build(cls, app: App, settings: Settings=Settings({}), base_url: http.URL=None):
         routes = app.routes
         url = get_schema_url(routes, base_url)
         content = get_schema_content(routes)
-        return cls(url=url, content=content)
+        schema = settings.get("SCHEMA", {})
+        title = schema.get("TITLE", None)
+        description = schema.get("DESCRIPTION", None)
```

Comment:
Probably prefer single quotes where possible (no particular reason, just prefer that stylistically)